.gitignore

```diff
@@ -122,3 +122,9 @@ Doc/api/*.rst
 
 #Ignore mypy cache directory
 .mypy_cache
+
+#Ignore virtual environment file
+.env
+
+#Ignore keys for publishing docs
+biopython_doc_key*
```

Comment:
Should probably add ``biopython_doc_key.pub`` too, but this hardly seem worth doing to me. This should only exist on one developer's machine - and most likely won't even be in this directory (I used to use ``~/.ssh/`` for this).

Reply:
i.e. Ignore both key files, or don't add either of them.